project/src/main/scala/org/broadinstitute/clio/integrationtest/ClioIntegrationTestSettings.scala

```diff
@@ -1,42 +1,90 @@
 package org.broadinstitute.clio.integrationtest
 
+import sbt._
 import sbt.Def.{Initialize, Setting}
 import sbt.Keys._
-import sbt._
 
 /**
-  * Settings definitions for the plugin.
+  * Settings for running our integration tests.
   */
-object ClioIntegrationTestSettings extends AutoPlugin {
-
-  /** The task key for running the integration tests. */
-  val testDocker: TaskKey[Unit] =
-    taskKey[Unit]("Run clio integration tests via docker.")
-
-  /** The task key for running the integration tests in Jenkins. */
-  val testDockerJenkins: TaskKey[Unit] =
-    taskKey[Unit]("Run clio integration tests via docker in Jenkins.")
-
-  private def creatTestTask(jenkinsTest: Boolean) = {
-    Def.task {
-      // Use the settings classDirectory for tests, and the project version
-      val testClassesDirectory = (classDirectory in Test).value
-      val clioVersion = version.value
-
-      // Wait for task test:compile, and the task to create the logs
-      (compile in Test).value
-      val log = streams.value.log
-
-      new ClioIntegrationTestRunner(testClassesDirectory,
-                                    clioVersion,
-                                    jenkinsTest,
-                                    log)
-        .run()
-    }
+object ClioIntegrationTestSettings extends ClioIntegrationTestKeys {
+
+  /** Directory to which all container logs will be sent during Dockerized integration tests. */
+  lazy val itLogDir: Initialize[File] = Def.setting {
```

Comment:
Since these fields are already scoped inside `ClioIntegrationTestSettings` it doesn't seem like they need an `it` name as well.